BUILD

```diff
@@ -0,0 +1,11 @@
+package(default_visibility = ["//visibility:public"])
+
+java_library(
+    name = "grpc_java",
+    exports = [
+        "//core",
+        "//stub",
+        "@com_google_code_findbugs_jsr305//jar",
```

Comment:
I would strongly prefer leaving the meta target out for now.  It's something we can easily add in later.

There are a couple reasons for this:
* Right now auto generated build files and tools that auto add deps have a hard time figuring out what to do when a file is exposed through multiple targets.  I think that making life easier for tools is a better long term strategy than making life easier for humans. (the former is a better deal for the latter).
* Bazel is not currently smart enough to trim deps not actually needed.  For example, IIRC `core` depends on the Census library.   Even if users really only needed `stub`, they would pull in Census too (I realize that stub depends on core, but that's a much easier thing to fix).  Same with the auth library, which depends on Google client libraries.  (this had the sad effect of pulling in additional licences for hundreds of Android apps that are infeasible to back out, despite none of them actually needing it)
* Bazel cannot figure out which deps are needed for compilation.  It knows if they are missing, but it cannot decide if they happened to be unnecessary.  This bloats jars.  There have been at least 5 or 6 tools internally that try to find unnecessary deps, but in practice they each failed on a different set of corner cases, and have a pretty high rate of being ignored.  

The temptation is strong to put everything in the `grpc_java`, especially for new deps.  (this happened for grpc's predecessor, and led to no jar being smaller than 200MB.)  Let's stay lean and mean for now, and add this target back in once there is strong evidence that it would be widely useful.